packages/components/package-lock.json

```diff
@@ -5035,11 +5035,6 @@
         }
       }
     },
-    "@chanzuckerberg/czedi-kit-tokens": {
-      "version": "0.0.1-alpha.6",
```

Comment:
lerna does not seem to want this entry here (running a bootstrap deletes it). Also, I think it was out of sync with the actual version).

Not having it here makes sense to me, since the package-lock.json is not published, and is only needed for local development. Lerna is taking care of setting up components' node_modules to have tokens.